pipelines/common.yaml

```diff
@@ -149,8 +180,6 @@ spec:
           value: $(params.output-image).prefetch
         - name: ociArtifactExpiresAfter
           value: $(params.image-expires-after)
-        - name: dev-package-managers
```

Comment:
Need to keep `dev-package-managers`

Reply:
It seems the `dev-package-managers` is not included in the latest konflux multiplatform build template, is this a paramater needed specifically by 'global-hub' case?

Reply:
We may want to add a line of comment for later pipeline maintainer.

Before me and @zhujian7 thought upgrade common would be the same like copy the last konflux pipeline template to each files. Now it seems be more complex then we thought.

I want to doc somewhere about the diff betweem our pipeline and konflux's template.

Reply:
We can directly set `dev-package-managers` in pipelineRun, no need to add an layer of params in pipeline file.

The details in: https://docs.google.com/document/d/13pWHhUx9NZwi3xnujkI7FxPpFkkfp9HXvjzCW4kiOOo/edit?usp=sharing